src/main/java/org/openelisglobal/dataexchange/fhir/service/FhirTransformServiceImpl.java

```diff
@@ -165,8 +166,7 @@ public class FhirTransformServiceImpl implements FhirTransformService {
     @Autowired
     private OrganizationService organizationService;
     @Autowired
-    @Qualifier("clientRegistryFhirClient")
-    private IGenericClient crClient;
```

Comment:
Removed this autowiring and the creation of the spring bean `clientRegistryFhirClient` because it was failing the application startup especialy in instances where it is working in isolation hence no connection to a fhir client e.g `http://openhim-core:5001/CR/fhir`